plugins/debug.cpp

```diff
@@ -511,6 +511,8 @@ struct FilterManager : public std::map<size_t, Filter>
     DebugManager::categorySignal_t::Connection connection_;
 };
 
+constexpr const char* FilterManager::configPath;
```

Comment:
TODO: verify that the initializer in the class above still runs

```c++
    constexpr static const char* configPath{"dfhack-config/runtime-debug.json"};
```

Reply:
Pretty sure it's fine based on https://github.com/DFHack/dfhack/blob/7b2cb8f9a631b8b44bfc2ff1f4e684d34b18b5f6/library/modules/MapCache.cpp#L82 and https://github.com/DFHack/dfhack/blob/7b2cb8f9a631b8b44bfc2ff1f4e684d34b18b5f6/library/include/modules/MapCache.h#L58